# Ejercicios

#### 1 Ejercicio

Para calcular el importe de IVA de  145 Euros es necesario multiplicar 145 por 0.21.
Escribe en Python la expresión que calcula el IVA de 145 Euros.

In [1]:
# Sol:
145*(21/100)

# El resultado es 30.45

30.45

Escribe en Python la expresión que calcula el IVA de 258 Euros.

In [2]:
# Sol:
258*(21/100)

54.18

Escribe una función en Python llamada __calcula_iva__ que calcule el importe de IVA de una cierta cantidad $x$.

In [4]:
# Sol
def calcula_iva(a):
    unIva=a*(21/100)
    return unIva

In [5]:
calcula_iva(9999)

2099.79

Ejecuta la función __calcula_Iva__ para las cantidades 145 y 258. Comprueba que los resultados son 30.45 y 54.18

In [7]:
calcula_iva(145)

30.45

In [8]:
calcula_iva(258)

54.18

#### 2 Ejercicio

Evalúa el polinomio $x^4 + x^3 + \frac{1}{2} x^2 - x$
en $x = 10$. El resultado
ha de ser  un número real.

In [13]:
# Sol:
def evalua(x):
    res=(x**4)+(x**3)+(0.5*(x**2))-x   ## pongo 0.5 en vez 1/2 para forzar casting a float
    return float(res)
# El resultado es 11040.0
evalua(10)

11040.0

In [15]:
evalua(7)

2761.5

Evalúa  el polinomio anterior para $x = 7$.

In [14]:
# Sol:
evalua(7)

2761.5

#### 3 Ejercicio

Define una función llamada __raiz_cubica__ que devuelva el valor de $\sqrt[3]{x}$. Recuerda que  $\sqrt[3]{x}$ = ${x}^{\frac{1}{3}}$


In [17]:
# Sol:
# El resultado de raiz cúbica de 5 es: 1.7099759466766968
def raiz_cubica (x):
    res=x**(1/3)
    return res    

In [19]:
raiz_cubica(5)

1.7099759466766968

Escribe la expresión Python que construya la siguiente cadena:

$$"El \ resultado \ de  \  la  \  raíz  \  cúbica  \   de  \ 5 \  es:  1.70"$$

Utiliza el operador de formato __%f__.

In [25]:
x=5
unaCadena='" El resultado de la raíz cúbica de %.2f es : %.2f "' % (x,raiz_cubica(x))
print(unaCadena)

" El resultado de la raíz cúbica de 5.00 es : 1.71 "
